apps/backend/src/modules/categories/category.service.ts

```diff
@@ -112,6 +113,23 @@ class CategoryService implements Service {
 		return { items };
 	}
 
+	public async findById(
+		query: CategoriesGetByIdRequestDto,
+	): Promise<CategoriesGetAllResponseDto> {
```

Comment:
Needs a new type here

Reply:
GetAll should be a list, findById I believe should return 1 item. We should have a type for 1 item and use it in GetAll type